src/FluentValidation.Tests.Benchmarks/Models.cs

```diff
@@ -69,10 +69,6 @@ public class NestedModel {
 	}
 

```

Comment:
I forget why I had to remove this, but it doesn't seem to be needed as far as I could tell